tests/testthat/test-sec-axis.R

```diff
@@ -0,0 +1,45 @@
+context("sec-axis")
+
+x <- exp(seq(log(0.001), log(1000), length.out = 100))
+foo <- data.frame(
+  x = x,
+  y = x/(1+x)
+)
+
+test_that("dup_axis() works", {
+  vdiffr::expect_doppelganger(
+    "dup_axis",
+    ggplot(foo, aes(x, y)) +
+      geom_point() +
+      scale_x_continuous(name = "Unit A",
```

Comment:
Better to set name to `NULL` here, since it's not germane to the test.  

I'd also consider using `geom_blank()` to help focus on the important part of the visual test